### DISCUSSION TITLE
Feature Request**: Allow `Tab` Key Indentation in Description Text Area

### DISCUSSION THREAD

**Discussion by tristdrum** [community/community#127067] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

**Description**:
When writing PR descriptions or comments in GitHub, the `Tab` key currently moves focus to the next form element. (currently "Markdown is supported") However, when editing Markdown or code within the description text area, it would be very useful if the `Tab` key could be used for indentation purposes (as it does in many code editors). This would improve the editing experience significantly.

**Current Behavior**:
- Pressing the `Tab` key moves the focus out of the text area.
- `Shift + Tab` does not work for outdenting text. (moves to the previous component, currently "Heading" in the toolbar)
- Indentation with the `Tab` key does not align bullet points correctly in Markdown lists.

**Proposed Behavior**:
- Pressing the `Tab` key within a text area should insert a tab character or spaces for indentation.
- Pressing `Shift + Tab` should outdent the text.
- Indentation should properly align bullet points in Markdown lists.

**Benefits**:
- Improved user experience when editing PR descriptions, comments, and other text areas.
- Enhanced productivity and ease of use for developers and collaborators using GitHub.

**Example**:
Similar functionality can be found in code editors like Visual Studio Code, where the `Tab` key is used for indentation within text areas.

Thank you for considering this feature request. This improvement would greatly enhance the usability of GitHub for all users.

Showcase:
https://github.com/community/community/assets/27028099/c677a7e0-826d-4f80-9346-efd4b4151a82

Device
- Macbook Pro M1 Pro

Browsers
- Chrome
- Safari

**Comment by ysl2** [comment]
+1

**Comment by barryZZJ** [comment]
As it turns out, select the text then press `Tab` will give you what you want. Not exactly what you wanted though, but at least it works.

**Comment by ThermodynamicBeta** [comment]
It's insane that it's 2026 and this basic QOL isn't implemented
